target/linux/generic/backport-6.1/738-net-phy-bcm54612e-add-suspend-resume.patch

```diff
@@ -0,0 +1,27 @@
+From 380b50ae3a04222334a3779b3787eba844b1177f Mon Sep 17 00:00:00 2001
```

Comment:
If this is a backport, prepend the kernel version this commit first appears in. (Filename)

```
738-vx.y-net-phy-bcm54612e-add-suspend-resume.patch
```

Reply:
done